src/sentry/options/defaults.py

```diff
@@ -408,6 +408,11 @@
 #       given fraction of orgs even if the corresponding feature flag is disabled.
 register("relay.transaction-metrics-org-sample-rate", default=0.0)
 
+# Sample rate for opting in orgs into the new transaction name handling.
+# old behavior: Treat transactions from old SDKs as high-cardinality.
+# new behavior: Treat transactions from old SDKs as low-cardinality, except for browser JS.
+register("relay.transaction-names-client-based", default=0.0)
```

Comment:
This option is added in a separate PR: https://github.com/getsentry/sentry/pull/37104.